storage/storage.go

```diff
@@ -36,6 +36,9 @@ type Storer interface {
 	GetReadmeMessage() (message string, err error)
 	FetchReadmeMessage(gistID string, gistFile github.GistFilename) (message string, err error)
 	UpdateReadmeMessage(session *discordgo.Session, original *discordgo.Message, upstream string) (err error)
+
+	PullWiki(wikiURL string) (err error)
```

Comment:
Since this storer interface is only used for database stuff, I think it would make more sense to create a _new_ interface specifically just for the wiki - this separates the concerns in a clearer way and doesn't add a MongoDB dependency to the wiki implementation.

Reply:
Oh I see 👍 I'll make new interface for that. I guess I've to remove it from [Memory](https://github.com/dakyskye/cj/blob/wiki-rework/storage/memory.go#L68-L73) interface because it seems it's used to fake database related functions.